aws-deadline/deadline-cloud-for-after-effects
# feat:Dockable submitter

### What was the problem/requirement? (What/Why)
We want to have a submitter with only aerender to avoid issues from adaptor.
### What was the solution? (How)
Pushed the branch with aerender only dockable submitter
### What is the impact of this change?
unblock more customers
### How was this change tested?
manual test locally by submitting jobs

----

*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*
